README.md

```diff
@@ -136,6 +136,34 @@ compile "com.android.support:appcompat-v7:$BAR"
 compile "com.android.support:palette-v7:$BAR"
 ```
 
+NOTE :
+The support libraries are now available through Google's Maven repository, so ensure that you have that added to your project's repositories.
```

Comment:
Let's change this to mention the specific version when things changed.  Something like "as of version `25.4.0` the support libraries are available only through Google's Maven repository..."